lib/model/rwfolder.go

```diff
@@ -1521,6 +1535,31 @@ func (p *rwFolder) moveForConflict(name string) error {
 	return err
 }
 
+// This function searches through both the current and newer version vector
+// arrays, every time there is a conflict.  What it expects to find is that one
+// of the versions will always be incremented by 1 from the previous version
+// list and the ID for that element is the one that made the change.  Thus its
+// ID is converted to binary and base32 encoded to find the first few chars and
+// those are returned.
+func (p *rwFolder) findNewestUpdateID() string {
+	var newestClientID uint64
+
+	for i := 0; i < len(p.newVersion); i++ {
+		// If the new version value has iterated even once from the previous
+		// version, then we know that's the guy who made the change.
+		if p.curVersion[i].Value < p.newVersion[i].Value {
+			newestClientID = p.newVersion[i].ID
```

Comment:
`break` I guess, as we'll be happy with just the first difference (although there may be multiple)